<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DLG_PREDEFINEDMBD</class>
 <widget class="QDialog" name="DLG_PREDEFINEDMBD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTreeWidget" name="TW_PreDefined_MBD">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="ButtonLayout">
    <item>
     <widget class="QPushButton" name="PB_Ok">
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="PB_Cancle">
      <property name="text">
       <string>Cancle</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
